<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Bouton_square">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Carre</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Battery : </string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Gauche :</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Droite :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_0">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; text-decoration: underline;&quot;&gt;Sensors :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_Gsens">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_Dsens">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_battery">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Bouton_Stop">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>160</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bouton_Tete">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>177</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Tete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bouton_Total">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Total</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bouton_Epaule">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Epaule</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bouton_Coude">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>170</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Coude</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bouton_Poignet">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Poignet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bouton_battery">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Battery</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bouton_sensors">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Sensors</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bouton_main">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Mains</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bouton_Camera">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>678</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Bouton_Stop</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>474</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>484</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuitter</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
